.github/workflows/code-quality.yaml

```diff
@@ -27,3 +27,32 @@ jobs:
 
       - name: Run ESLint
         run: cd src && composer run eslint
+
+  selenium-tests:
+    name: Selenium Tests
+    runs-on: ubuntu-latest
+    env:
+      # ECR_BACKEND_REPOSITORY: organic-platform-backend
+      PROJECT: wordpress-plugin
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Build the stack
+        id: build-stack
+        run: |
+          docker compose -p ${{ env.PROJECT }} \
```

Comment:
Doh! Docker isn't installed here. Note to self: reference https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action